jigglypuffRL/common/base.py

```diff
@@ -15,22 +15,22 @@ def __init__(self, discrete, deterministic, **kwargs):
         self.model = None
 
     def forward(self, state):
-        return self.model(state)
+        return self.model.forward(state)
```

Comment:
Why is this necessary?

Reply:
Not necessary at all, was trying to debug through some errors. So just though explicitly stating it would be better. 

We can reset this change though

Reply:
This might cause errors, that is why I pointed this out.

Reply:
Why?

Reply:
https://discuss.pytorch.org/t/any-different-between-model-input-and-model-forward-input/3690/4

Reply:
I know forward just uses `__call__` but why would this create errors?